scripts/config_check/refresh.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
```

Comment:
This needs `+x` bit set. Your squash pushes are very odd. The keep closing and reopening and generating multiple notifications. Just add a single normal commit next time.

Reply:
huh. not even touching that file in Windows after setting the `-x` in a VM seems to clear that bit upon re-adding files in a commit in WSL.

Reply:
It seems that when I reset my branch and force push prior to recommitting squashed changes, GitHub sees that there are no changes in my branch and thus automatically sets the PR to "merged" and closes it. Not at all desired nor intuitive.

Reply:
As I think if this more, I agree. I'll add this text to the README.md for the output files:

These files are all included in the GitHub `.ignore` file in the root of the repository. 
Edit that file or redirect the output to a different location if you'd like to track changes.

Reply:
regarding the `+x` that keeps getting lost in WSL: [stackoverflow](https://stackoverflow.com/questions/68281828/how-can-i-keep-the-execution-permissions-when-cloning-a-repository-in-windows) to the rescue (see near the end _"Manually updating modes when your file system does not cooperate_":  

```
 git add refresh.sh --chmod=+x
```